core/src/stores/route.ts

```diff
@@ -163,12 +165,17 @@ const routeInfoStore = createStore({
 
       curUrlFull.add(urlKey);
 
+      const markedRoutePreview = {
+        ...state.markedRoutePreview,
+        ...(currentRoute?.searchMark ? { [currentRoute.searchMark]: search } : {}),
```

Comment:
this cache may exist forever, would it cause problem?

Reply:
it's only use for back to up